README.md

```diff
@@ -2,12 +2,217 @@
 
 Publish your translations faster and simpler than ever before. Stop waiting for the next deployment and start publishing all your translations in real-time directly in Phrase Strings.
 
+Head over to the Phrase Help Center to learn about this feature and how to use it in your apps: https://support.phrase.com/hc/en-us/articles/5804059067804
+
 ## Instructions
 
-Head over to the Phrase Help Center to learn about this feature and how to use it in your apps: https://support.phrase.com/hc/en-us/articles/5804059067804
+With the SDK, the app regularly checks for updated translations and downloads them in the background.
 
----
+[Mac Catalyst](https://developer.apple.com/mac-catalyst) is also supported.
+
+The SDK can be installed manually or via Swift Package Manager, Carthage or Cocoa Pods.
+
+Example [app](https://github.com/phrase/ios-demo-app)
+
+### Swift Package Manager
+Add the public repository URL (https://github.com/phrase/ios-sdk/). Xcode automatically handles the rest of the installation.
+
+### Carthage
+Add the following line into your Cartfile:
+
+```
+binary "https://raw.githubusercontent.com/phrase/ios-sdk/master/PhraseSDK.json" ~> 3.0.0
+```
+
+Run `carthage update` and add the `PhraseApp.framework` to your project as described in the [Carthage documentation](https://github.com/Carthage/Carthage#if-youre-building-for-ios-tvos-or-watchos).
 
+### Cocoa Pods
+Add the following line into your Podfile:
+
+```
+pod 'PhraseSDK'
+```
+Run `pod install`. If new to CocoaPods, see their [documentation](https://guides.cocoapods.org/using/getting-started.html).
+
+### Manual installation
+
+1. Download the latest [release](https://github.com/phrase/ios-sdk/releases).
+2. Add PhraseSDK.framework in Xcode as the linked binary to the target.
+3. A script to strip the extra binaries needs to be run before you upload the app as the Apple store rejects apps including simulator binaries.
```

Comment:
The run script is no longer be necessary, because of xcframework

Reply:
Ah good to know!